### PR TITLE
Change KMeans "mus" parameter to "cluster_centers"

### PR DESCRIPTION
**Pull request by vinnik-dmitry07**
Tada
──────────▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
────────█═════════════════█
──────█═════════════════════█
─────█════════▄▄▄▄▄▄▄════════█
────█════════█████████════════█
────█═══════██▀─────▀██═══════█
───███████████──█▀█──███████████
───███████████──▀▀▀──███████████
────█═══════▀█▄─────▄█▀═══════█
────█═════════▀█████▀═════════█
────█═════════════════════════█
────█══════▀▄▄▄▄▄▄▄▄▄▀════════█
───▐▓▓▌═════════════════════▐▓▓▌
───▐▐▓▓▌▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▐▓▓▌▌
───█══▐▓▄▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄▓▌══█
──█══▌═▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌═▐══█
──█══█═▐▓▓▓▓▓▓▄▄▄▄▄▄▄▓▓▓▓▓▓▌═█══█
──█══█═▐▓▓▓▓▓▓▐██▀██▌▓▓▓▓▓▓▌═█══█
──█══█═▐▓▓▓▓▓▓▓▀▀▀▀▀▓▓▓▓▓▓▓▌═█══█
──█══█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓█══█
─▄█══█▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌█══█▄
─█████▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌─█████
─██████▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌─██████
──▀█▀█──▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌───█▀█▀
─────────▐▓▓▓▓▓▓▌▐▓▓▓▓▓▓▌
──────────▐▓▓▓▓▌──▐▓▓▓▓▌
─────────▄████▀────▀████▄
─────────▀▀▀▀────────▀▀▀▀

### CODE REVIEW

**Comment by vigsterkr** [comment]
ansi art FTW \o/

File: doc/ipython-notebooks/clustering/KMeans.ipynb [hunk 1]
```diff
@@ -288,7 +288,7 @@
     "new_initial_centers = array([[5.,5.],[0.,100.]])\n",
     "\n",
     "# set new initial centers\n",
-    "kmeans.put(\"mus\", new_initial_centers)"
+    "kmeans.put(\"cluster_centers\", new_initial_centers)"
```
Comment (gf712) [inline doc/ipython-notebooks/clustering/KMeans.ipynb:291]: @vigsterkr I think there should be some sort of check when using put here. Also this should be done with a .put(“initial_cluster_centers”) and .put(“cluster_centers”) should be READONLY/watch_method.

**Review by gf712 — COMMENTED** [review]
Just a comment, doesn’t have to be done in this PR

File: doc/ipython-notebooks/clustering/KMeans.ipynb (hunk 1 above)
Comment (vigsterkr) [reply doc/ipython-notebooks/clustering/KMeans.ipynb:291]: yep second that!

File: src/shogun/clustering/KMeansBase.cpp [hunk 2]
```diff
@@ -321,7 +321,7 @@ void KMeansBase::init()
 	SG_ADD(
 	    &use_kmeanspp, "kmeanspp", "Whether to use kmeans++",
 	    ParameterProperties::HYPER | ParameterProperties::SETTING);
-	SG_ADD(&mus, "mus", "Cluster centers", ParameterProperties::MODEL);
+	SG_ADD(&cluster_centers, "cluster_centers", "Cluster centers", ParameterProperties::MODEL);
```
Comment (vigsterkr) [inline src/shogun/clustering/KMeansBase.cpp:324]: as @gf712 suggested this should be READONLY.
and there should be another parameter `initial_cluster_centers` that actually holds the cluster centers that can be provided by the user

**Review by vigsterkr — CHANGES_REQUESTED** [review]

**Comment by vinnik-dmitry07** [comment]
Closed due to https://github.com/shogun-toolbox/shogun/pull/4962

**Pull request closed without merging**